src/pages/components/autocomplete.mdx

```diff
@@ -1,10 +1,110 @@
 ---
 title: Autocomplete
 description: >
-  Description will be here
+  Autocomplete is an input field that filters results as users type. This helps users find
+  something quickly in a large list of options.
+package: '@zendeskgarden/react-dropdowns'
+components:
+  - dropdowns/src/elements/Autocomplete/Autocomplete.tsx
+  - dropdowns/src/elements/Dropdown/Dropdown.tsx
+  - dropdowns/src/elements/Menu/Menu.tsx
+  - dropdowns/src/elements/Menu/Items/Item.tsx
+  - dropdowns/src/elements/Fields/Field.tsx
+  - dropdowns/src/elements/Fields/Hint.tsx
+  - dropdowns/src/elements/Fields/Label.tsx
 ---
 
-Content here
+<Usage>
+  <Use>
+    <ul>
+      <li>To filter down a large list of options</li>
+      <li>To quickly find a known option</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        To make more than one selection use <a href="./multiselect">Multiselect</a> instead
+      </li>
+    </ul>
+  </Misuse>
+</Usage>
+
+## How to use it
+
+### Default
+
+The basic usage of an Autocomplete component.
+
+import Default from '../../examples/dropdowns/autocomplete/Default.tsx';
+import DefaultCode from '!!raw-loader!../../examples/dropdowns/autocomplete/Default.tsx';
+
+<CodeExample code={DefaultCode}>
+  <Default />
+</CodeExample>
+
+### Size
+
+Autocomplete comes in two sizes: regular (default) and compact.
+
+import Size from '../../examples/dropdowns/autocomplete/Size.tsx';
+import SizeCode from '!!raw-loader!../../examples/dropdowns/autocomplete/Size.tsx';
+
+<CodeExample code={SizeCode}>
+  <Size />
+</CodeExample>
+
+### Media
+
+Media elements add even more context to an Autocomplete.
+
+import Media from '../../examples/dropdowns/autocomplete/Media.tsx';
+import MediaCode from '!!raw-loader!../../examples/dropdowns/autocomplete/Media.tsx';
+
+<CodeExample code={MediaCode}>
+  <Media />
+</CodeExample>
+
+## How to use it well
+
+### Use Autocomplete to narrow down options
+
+Autocomplete is useful for lists that are too long to read in full (for example, company employees)
+or that accumulate items over time (for example, inventory).
```

Comment:
@rossmoody Will we have a visual for this example?

Reply:
Updated this with a visual example 👍 .